web-components/svelte-react.ts

```diff
@@ -23,20 +23,21 @@ export type SvelteProps<T> = T extends SvelteComponentTyped<infer Props, any, an
 export type SvelteEvents<T> = T extends SvelteComponentTyped<any, infer Events, any> ? Events : {}
 export type ReactProps<Props, Events> = Props & {
   [P in keyof Events as `on${Capitalize<P & string>}`]?: (e: Events[P]) => void
-} 
+} & {
+  ref?: ForwardedRef<Props>
+}
 
 /**
  * 
  * @param tag custom element tag name for svelte component
  * @param component The imported svelte component itself. This is not used, but ensures that the component's code has been included in the bundle.
  * @returns A react component
  */
-export default function SvelteWebComponentToReact<T extends Record<string, any>> (tag: string, component: typeof SvelteComponent) {
-  console.log(component)
-  return function ReactSvelteWebComponent (props: React.PropsWithChildren<T>) {
+export default function SvelteWebComponentToReact<T extends Record<string, any>>(tag: string, component: typeof SvelteComponent) {
+  return forwardRef((props: PropsWithChildren<T>, forwardedRef: ForwardedRef<SvelteComponent>) => {
```

Comment:
Made it possible to get a ref to the current component, like on any other html element. This makes it easier to wire up scroll handlers without making React components rerender all the time.